recurring_contract/model/recurring_contract.py

```diff
@@ -60,24 +59,16 @@ class recurring_contract(models.Model):
     _description = "Contract for recurring invoicing"
     _inherit = ['mail.thread']
     _rec_name = 'reference'
-
-    @api.depends('contract_line_ids', 'contract_line_ids.amount',
-                 'contract_line_ids.quantity')
-    def _get_total_amount(self):
-        self.total_amount = sum([line.subtotal for line in
-                                 self.contract_line_ids])
 
-    @api.one
-    def _get_last_paid_invoice(self):
-        self.last_paid_invoice_date = max(
-            [invl.invoice_id.date_invoice for invl in self.invoice_line_ids
-             if invl.state == 'paid'] or [False])
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
 
     reference = fields.Char(
         _('Reference'), default="/", required=True, readonly=True,
         states={'draft': [('readonly', False)]})
-    start_date = fields.Date(
-        _('Start date'), default=datetime.today().strftime(DF),
+    start_date = fields.Date(
+        _('Start date'), default=datetime.today().strftime(DF),
```

Comment:
Translation function not useful for field names. Field names not useful if same as field_name.